tests/test_FindTest.py

```diff
@@ -190,122 +200,112 @@ def test_find_citations(self):
             ('before asdf, 1 U. S., at 2',
              [case_citation(2, page='2', reporter='U.S.',
                             reporter_found='U. S.', short=True,
-                            antecedent_guess='asdf,')]),
+                            antecedent_guess='asdf')]),
             # Test short form citation with preceding ASCII quotation
             ('before asdf,” 1 U. S., at 2',
              [case_citation(2, page='2', reporter_found='U. S.',
-                            short=True, antecedent_guess='asdf,”')]),
+                            short=True)]),
```

Comment:
Why is there no longer an antecedent detected for this one?

Reply:
The antecedent word now has to match `([\w\-.]+),?`.

Possibly we'll find other things that have to be added to that, but I think it's correct not to include trailing quotes -- `asdf` isn't the short name of 1 U.S. in that test, is it?

Reply:
Ah yes, I see, that makes sense!